andela/eventx
# Ft event find insights 129622087

#### What does this PR do?

Creates a feature that captures and presents insights on how an event was found.
#### Description of Task to be completed?

As an event manager I can get actionable insights regarding the channels through which users found my event.
#### How should this be manually tested?

Clone repository and start the server. 
Create an event manager profile and at least one event
Create a user profile and attend your event, an option requesting how event was found is now available.
Go to your event manager dashboard and check for the bookings, a field now shows how the event was found.
#### Any background context you want to provide?

Not really
#### What are the relevant pivotal tracker stories?
#129622087
#### Screenshots (if appropriate)
#### Questions:
